01_login.php

```diff
@@ -73,29 +95,46 @@
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-padding w3-card">
+				<div class="w3-left">
 				<img src="images/placeholder.jpg" width="40" height="40">
+				</div>
 				<a class="w3-bar-item"><b>Soundwave</b> </a>
 			</div>
 		</div>
+		
 
 
 		
 		<!-- background container; colour orange -->
-		<div class="container2" style="max-width:100%">
-		
+		<div class="w3-orange container w3-image">
+			<div class="bg">
+
+
+				<br>
+				<br>
+				<br>
+				<br>
+
+
+				<center>
+					<div class= "container w3-panel w3-padding-large-16" style="height:80%;width:80%; background-color:rgba(255, 255, 255, .2);">
```

Comment:
added a clear container to keep content together; using the colour of a white shade - background-color:rgba(255, 255, 255, .2)